CMakeLists.txt

```diff
@@ -33,12 +33,11 @@ IF (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 	SET (CMAKE_INSTALL_PREFIX ${FESAPI_BINARY_DIR}/install CACHE PATH "FesapiCpp install prefix" FORCE)
 ENDIF (CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT)
 
+set(CMAKE_CXX_STANDARD 11)
```

Comment:
From cmake doc : "a compiler which does not support -std=gnu++11 or an equivalent flag will not result in an error or warning, but will instead add the -std=gnu++98 flag if supported"
I think it is not strictly equivalent with "-std=c++0x".
This pull request is supposed to give up support for pre-C++11 compiler. So, my question is just for information.

Reply:
you are exactly right

Reply:
Would it be possible to raise a cmake error if C+11 is not supported by the compiler?

Reply:
The CMAKE_CXX_STANDARD_REQUIRED should do the job (see https://crascit.com/2015/03/28/enabling-cxx11-in-cmake/)  but I never tested it

Reply:
OK thanks!